src/winter_onboarding_2021/fleet_management_service/views/invite.clj

```diff
@@ -0,0 +1,69 @@
+(ns winter-onboarding-2021.fleet-management-service.views.invite
+  (:require [winter-onboarding-2021.fleet-management-service.views.core :as core]))
+
+(def headers
+  [{:label "Token" :value :invites/token}
+   {:label "Role" :value :invites/role}
+   {:label "Valid Until" :value :invites/valid-until}
+   {:label "Usages Limit" :value :invites/usages-done}
+   {:label "Status" :value :invites/status}])
+
+(defn invite-form []
+  [:div.col-md-6
+   [:aside
+    [:p "Generate a new token:"]
+    [:div {:class "card"}
+     [:article {:class "card-body"}
+      [:form {:action "/invites" :method "POST"}
+       [:div {:class "form-group"}
+        [:label {:class "form-label"} "Select Role:"]
+        [:select
+         {:name "role" :class "form-select"}
+         [:option {:value "admin"} "Admin"]
+         [:option {:value "manager"} "Manager"]
+         [:option {:value "driver"} "Driver"]]]
+       (core/labelled-text-input "Number of accounts:"
+                                 :require true
+                                 :name "usage-limit"
+                                 :type "number"
+                                 :max 50
+                                 :min 1)
+       (core/labelled-text-input "Valid Until:"
+                                 :require true
+                                 :name "valid-until"
+                                 :type "date")
+       [:div
+        [:button {:class "btn btn-primary btn-block"
+                  :type "submit"}
+         "Create Token"]]]]]]])
+
+(defn format-date [date]
+  (.format (java.text.SimpleDateFormat. "MM/dd/yyyy") date))
```

Comment:
Utils also has a format-date function. Might be better if we just name it something like `mm-dd-yy`.

Or much better if we have a function like `format-date` which accepts a `date` object & the format string like `MM/dd/yyyy`